fsm_team_handler/models/fsm_person_extended.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, fields, models
+
+
+class EmployeeAvailability(models.Model):
+    _inherit = 'fsm.person'
+
+    available = fields.Boolean(
+            string="Availability",
+            default=True,
+            help="Shows status of the person."
+                 "This status will be updated "
+                 "when person is started/finished "
+                 "a job."
+    )
+    # skill sets of this person
+    skill_ids = fields.Many2many(
+            'fsm.skills',
+            'person_skills_rel',
+            'person_id',
+            'skill_id',
+            string="Basic Skills"
+    )
+
+
+class ResPartnerPerson(models.Model):
+    _inherit = 'res.partner'
+
+    @api.multi
+    def write(self, vals):
```

Comment:
I think we don't need this method at all.

Reply:
That's already described in the specs and implemented: 
there is a Wizard to convert a Parnter into a Person: https://github.com/OCA/field-service/blob/11.0/fieldservice/wizard/fsm_wizard.py